src/static/scss/all.scss

```diff
@@ -8,6 +8,8 @@
 @import 'common/colors';
 @import 'common/typography';
 @import 'common/grid';
+@import 'common/reset';
```

Comment:
Please place the reset at the top, it should first reset and the get your styles. Also, is it an idea to load `normalize`?